jellyfin_mpv_shim/utils.py

```diff
@@ -230,10 +231,18 @@ def get_resource(*path):
     # Detect if bundled via pyinstaller.
     # From: https://stackoverflow.com/questions/404744/
     if getattr(sys, "_MEIPASS", False):
-        application_path = os.path.join(getattr(sys, "_MEIPASS"), "jellyfin_mpv_shim")
+        if platform.system() == "Darwin":
+            application_path = os.path.dirname(sys.executable)
+        else:
+            application_path = os.path.join(getattr(sys, "_MEIPASS"), "jellyfin_mpv_shim")
     else:
         application_path = os.path.dirname(os.path.abspath(__file__))
 
+    # ! Test code for Mac
+    if getattr(sys, 'frozen', False):
+        application_path = os.path.join(os.path.dirname(sys.executable), "../Resources")
```

Comment:
The condition doesn't seem to be just if Mac OS but would also apply to PyInstaller and break windows support. Should the condition be `platform.system() == "Darwin" and getattr(sys, 'frozen', False)`?

Also please remove the print statement or change it to a debug log message.

Reply:
Yes will do.

Reply:
This still looks like it isn't checking the platform?